foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -31,6 +31,7 @@ using OpCode = websocketpp::frame::opcode::value;
 static const websocketpp::log::level APP = websocketpp::log::alevel::app;
 static const websocketpp::log::level RECOVERABLE = websocketpp::log::elevel::rerror;
 
+constexpr size_t MAX_SEND_BUFF_SIZE_BYTES = 100000000;  // 100MB
```

Comment:
This can be queried from websocketpp, we don't need to hardcode it

Reply:
Ah nice, will do

Reply:
I can retrieve the max message size (https://github.com/zaphoyd/websocketpp/blob/1b11fd301531e6df35a6107c1e8665b1e77a2d8e/docs/config.dox#L87) but so far I haven't found a way to get the max send buffer size. It's probably deeper down in asio